ee/localserver/krypto-ec-middleware.go

```diff
@@ -259,14 +284,45 @@ func (e *kryptoEcMiddleware) Wrap(next http.Handler) http.Handler {
 		bhr := &bufferedHttpResponse{}
 		next.ServeHTTP(bhr, newReq)
 
-		var response []byte
+		response := response{
+			Nonce:     ulid.New(),
+			Timestamp: time.Now().UTC().Unix(),
+			Data:      bhr.Bytes(),
+		}
+
+		if runtime.GOOS == "darwin" {
+			requestDataWithUserSig, err := e.addUserSignature(r.Context(), response, *challengeBox)
```

Comment:
Should we just go ahead and add a UserSig or MachineSig for all platforms? We could just do the same behavior with TPM.